drag_and_drop_v2/templates/html/drag_and_drop_edit.html

```diff
@@ -116,6 +116,7 @@
                 <h3>{% trans "Items" %}</h3>
             </header>
             <section class="tab-content">
+                <div class="xblock-editor-error-messages" aria-live="polite" tabindex="0"></div>
```

Comment:
@cahrens Fixed aria-live and tabindex.

As for showing this div on all tabs - it only shows error feedback from backend, and this tab is the only one actually sending the feedback. Your suggestion totally makes sense, but it is not necessary now, so I would keep it where it is needed only.

To be honest, I don't like the editor much, so we might want to completely overhaul it (both design and UX) later. But that's my point of view :)